src/components/meeting/meetingTime/JitsiComponentWrapper.js

```diff
@@ -33,7 +33,7 @@ export const JitsiWrapper = (props) => {
         setJitsiWidth('20%')
         setActionComponent(
             <div style={{float: "left", width: '80%', height: '80vh', overflow: 'scroll'}}>
-                <TaskWrapper id={16} submitLeftOffset={'25%'}/>
+                <MultipleTasksWrapper id={16} submitLeftOffset={'25%'}/>
```

Comment:
Może głupie pytanie ale w którym tasku mamy zamiane tego id na jakiegoś propa?

Reply:
Ja to mam - znaczy już prawie mam - chyba w ES-210